Intersect.Server/Localization/CommandsNamespace.cs

```diff
@@ -363,6 +373,21 @@ public sealed class ArgumentsNamespace : LocaleNamespace
                     Description = @"the name of the player to unmute"
                 };
 
+                [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+                public readonly LocaleArgument VarGUID = new LocaleArgument
+                {
+                    Name = @"GUID",
+                    Description = @"The GUID of the server var to set"
+                };
+
+                [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+                public readonly LocaleArgument VarValue = new LocaleArgument
+                {
+                    Name = @"Value",
+                    Description = @"The value to set the server var to"
```

Comment:
-> `the value to set the server variable to`

Reply:
Done